files/routes/posts.py

```diff
@@ -603,7 +603,7 @@ def delete_post_pid(pid, v):
 	if post.author_id != v.id:
 		abort(403)
 
-	post.deleted_utc = int(time.time())
+	post.state_user_deleted_utc = datetime.now()
```

Comment:
This works as written, but see other comments on tz-aware datetimes.
Recommend using `datetime.now(tz=timezone.utc)`
Necessary imports if not already available: `from datetime import datetime, timezone`

Reply:
Should now be fixed.